spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:all) do
+    @user = User.new(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+  end
+
+  context 'User implementation testing' do
+    it 'User name should be present' do
+      expect(@user.name).to eq('John')
+    end
+
+    it 'Recent post method should return zero' do
+      expect(@user.three_most_recent_post.length).to be 0
+    end
+
```

Comment:
- [ ] 2 out of 8 tests are failing, Please fix this and make sure all tests are passing successfully.

![image](https://user-images.githubusercontent.com/83803414/204024241-fdb82229-1fae-45c0-ad0b-7d0481822fb1.png)

Reply:
Thank you very much for your comment, and sorry for the mistake. In this version, my functions had not been updated but I already fixed it.